src/index.html

```diff
@@ -26,7 +26,7 @@
     <div id="sidebar" class="no-highlight" onmousedown="setHighlight('mousedown', 'sidebar')"
     onmouseleave="setHighlight('mouseleave', 'sidebar')">
       <input type="checkbox" id="toggle" oninput="toggleHandler()" style="display: none;">
-      <label class="toggle-switch" for="toggle">
+      <label class="toggle-switch" for="toggle" style="display: block; margin:auto;">

```

Comment:
You cannot put CSS in line the HTML like this, it is extremely bad practice. It has to be placed in the CSS file.